public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts

```diff
@@ -65,6 +68,13 @@ export const Messages = {
         username: 'Username',
         password: 'Password',
       },
+      postgresqlDetails: {
+        address: 'Hostname',
```

Comment:
Here the same logic applies, these strings are already presented in `mainDetails`

Reply:
sure, fixed